hw/core-v-mini-mcu/core_v_mini_mcu.sv.tpl

```diff
@@ -39,19 +39,24 @@ ${pad.core_v_mini_mcu_interface}
     input  obi_req_t  [EXT_XBAR_NMASTER_RND-1:0] ext_xbar_master_req_i,
     output obi_resp_t [EXT_XBAR_NMASTER_RND-1:0] ext_xbar_master_resp_o,
 
+% if int(ao_peripherals_num_spc) > 0:
```

Comment:
this is probably better to do it with an SV parameter rather than hjson parameter as this is an external extensions

Reply:
I figured that its much easier to change it if it's the mcu_cfg.hjson, since this parameter is used in multiple files.

Reply:
If it needs to be a localparam, where could I put it in order to make it easly available to developers? It can't be in core_v_mini_mcu.sv since its generated by a template.

Reply:
where is it used?

Reply:
In core_v_mini_mcu_pkg.sv ao_peripheral_subsystem.sv, AO_SPC_NUM. This parameter is then used in ao_peripheral_subsystem.sv to generate the AO peripheral bus, in core_v_mini.sv to set the dimensions of the port "ext_ao_peripheral_slave_req_i" and respective resp_o, in x_heep_system for the same reason, as in testharness.sv

Reply:
if this is just a parameter that is not messing up the generation (in this case it does not) should be just an SV parameter and not a template, otherwise whoever eXtends X-HEEP has to modify the hjson... which is meant not for extensions but for configuration.

Reply:
Here is how I solved it: I created a new package called ao_spc_pkg.sv which contains the AO_SPC_NUM parameter used throughout X-Heep. In this way, I removed the field in mcu_cfg.hjson while still making it accessible from different files.